sphinx-doc/_templates/page.html

```diff
@@ -0,0 +1,71 @@
+{% extends "furo/page.html" %}
+{% block footer %}
+<div class="related-pages">
+{% if next -%}
```

Comment:
What is this file for?

Reply:
This is a Jinja template that replaces the `footer` block of furo's page https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/page.html. I use it to add the UMich logo and the statement: "Development of HOOMD-blue is led by the Glotzer Group at the University of Michigan (supported by NSF DMR 1808342) with many external contributions." on every page of the documentation. This was done in `footer.html` for `sphinx-rtd-theme`.